h2/connection.py

```diff
@@ -986,6 +1010,9 @@ def ping(self, opaque_data):
         f.opaque_data = opaque_data
         self._prepare_for_sending([f])
 
+        self.config.logger.debug("Sending ping frame")
```

Comment:
This log, and ones like it, should probably be at the top of the log, not the bottom.